src/Utilities/Compiler/Extensions/IMethodSymbolExtensions.cs

```diff
@@ -509,5 +509,29 @@ public static bool IsLockMethod(this IMethodSymbol method, INamedTypeSymbol syst
 
         public static bool HasParameterWithDelegateType(this IMethodSymbol methodSymbol)
             => methodSymbol.Parameters.Any(p => p.Type.TypeKind == TypeKind.Delegate);
+
+        /// <summary>
+        /// Find out if the method overrides from target virtual method of a certain type
+        /// </summary>
+        /// <param name="methodSymbol">The method</param>
+        /// <param name="typeSymbol">The type has virtual method</param>
+        public static bool IsOverrides(this IMethodSymbol methodSymbol, INamedTypeSymbol typeSymbol)
```

Comment:
>IsOverrides [](start = 27, length = 11)

"IsOverrideOrVirtual"?  "IsOverrideOrVirtualMethodOf"?  Because this would return true on the base virtual method (as it should).